src/Controls/src/Core/Xaml/TypeConversionExtensions.cs

```diff
@@ -197,6 +197,12 @@ internal static object ConvertTo(this object value, Type toType, Func<TypeConver
 						return TimeSpan.Parse(str, CultureInfo.InvariantCulture);
 					if (toType == typeof(DateTime))
 						return DateTime.Parse(str, CultureInfo.InvariantCulture);
+#if NET6_0_OR_GREATER
```

Comment:
this means you can have properties of type DateOnly in xaml and support parsing. There's no test showing that it works, so it's probably not needed, remove it

Reply:
Removed the changes.